scripts/all_checks.sh

```diff
@@ -43,17 +53,16 @@ EOF
     echo "-- venv created. --"
   fi
 
-  py3=$(readlink -f "${venv_dir}/bin/python3")
-  pylint=$(readlink -f "${venv_dir}/bin/pylint")
-  flake8=$(readlink -f "${venv_dir}/bin/flake8")
-  isort=$(readlink -f "${venv_dir}/bin/isort")
-  black=$(readlink -f "${venv_dir}/bin/black")
+  py3="${venv_dir}/bin/python3"
+  pylint="${venv_dir}/bin/pylint"
+  flake8="${venv_dir}/bin/flake8"
+  isort="${venv_dir}/bin/isort"
+  black="${venv_dir}/bin/black"
 
   echo "-- check copyright. --"
   ${py3} ./scripts/list_missing_copyright.py
 
-  if [ -z "${no_pylint}" ]
-  then
+  if [ -z "${no_pylint}" ]; then
```

Comment:
```suggestion
  if [ -z "${no_pylint}" ]
   then
```
@apaolillo Like this?

Reply:
yes

Reply:
I did the requested changes.